istioctl/cmd/dashboard.go

```diff
@@ -409,11 +412,16 @@ func closePortForwarderOnInterrupt(fw kube.PortForwarder) {
 		fw.Close()
 	}()
 }
-func openBrowser(url string, writer io.Writer) {
+func openBrowser(url string, writer io.Writer, skipOpen bool) {
```

Comment:
Rename this function.  It now does two things, either printing the URL or print+open.

Reply:
I think they are parts of one thing.